src/Core/Framework/Store/Command/StoreSkipFirstRunWizard.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Store\Command;
+
+use Shopware\Core\Framework\Adapter\Console\ShopwareStyle;
+use Shopware\Core\Framework\Context;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Shopware\Core\Framework\Store\Services\FirstRunWizardClient;
+
+class StoreSkipFirstRunWizard extends Command
```

Comment:
wouldn't it also make sense to add this as an option to the `system:install` command?

As in most cases you want to skip the first run wizard if you setup shopware over CLI with that command, so having an additional option there would be nice, internally the system install command calls a bunch of other commands, so you should be able to adapt that, to call also this command if a flag is passed